src/TouristInfoCards.js

```diff
@@ -0,0 +1,19 @@
+
+
+
+function TouristInfoCards (props){
+    return (
+        
+       <div className="card">
+       
+         <img src={props.imageSource} className="card-img-top" />
+            <div className="card-body">
+            <a href={props.link} className="btn btn-primary">{props.town}</a>
+           </div>
+           </div>
+           
+           );
+           }
```

Comment:
perfect use of a component, good job. Are you using `prettier` to format this? looks like the indentation has gone wonky